dashboard/lib/policies/lti.rb

```diff
@@ -90,11 +90,11 @@ module MessageType
     jwks_uri: CDO.studio_url('/oauth/jwks', CDO.default_scheme),
     token_endpoint_auth_method: "private_key_jwt",
     contacts: ["platform@code.org"],
-    scope: "https://purl.imsglobal.org/spec/lti-ags/scope/score https://purl.imsglobal.org/spec/lti-nrps/scope/contextmembership.readonly",
+    scope: ALL_SCOPES.join(' '),
     "https://purl.imsglobal.org/spec/lti-tool-configuration" => {
-      domain: "studio.code.org",
-      description: "Code.org",
-      target_link_uri: "https://studio.code.org/lti/v1/sync_course",
+      domain: CDO.studio_url('', CDO.default_scheme),
```

Comment:
Per the [LTI spec](https://www.imsglobal.org/spec/lti-dr/v1p0#lti-configuration-0) it looks like domain shouldn't include the protocol which I believe this change will add. 

> The primary domain covered by this tool; protocol must not be included. For example mytool.example.org

Reply:
Hmmm yeah you're right. I'll look into if we have a helper method that leaves the protocol off.

Reply:
Looks like I can just leave the second argument blank, and it defaults to an empty string.

Reply:
Looks like I should use [CDO.dashboard_site_host](https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo.rb#L114)

Reply:
Tested on multiple environments:

Prod
```
[production] dashboard > CDO.dashboard_site_host
=> "studio.code.org"
```

Test
```
[test] dashboard > CDO.dashboard_site_host
=> "test-studio.code.org"
```

Local development
```
[development] dashboard > CDO.dashboard_site_host
=> "localhost-studio.code.org:3000"
```